src/main/java/frc/robot/systems/ElevatorArmFSM.java

```diff
@@ -118,7 +120,10 @@ public void update(TeleopInput input) {
 		if (currentState != FSMState.IDLE) {
 			currentEncoder = armMotor.getEncoder().getPosition();
 		}
-		if (limitSwitchLow.isPressed()) {
+		if (limitSwitchLow.isPressed() && !lastPressed) {
+			armMotor.getEncoder().setPosition(0);
+			currentEncoder = 0;
+		} else if (!limitSwitchLow.isPressed() && lastPressed) {
```

Comment:
This is doing the same thing in both blocks. Simplify into one expression.

(a && !b) || (!a && b) = (a&&b) || (!a && !b)